src/bin/pgcopydb/ld_apply.c

```diff
@@ -981,9 +981,9 @@ stream_apply_sql(StreamApplyContext *context,
 			 *
 			 * [1] https://www.postgresql.org/docs/current/replication-origins.html
 			 */
-			char *sql = "SELECT txid_current()";
+			char *sqlCmd = "SELECT txid_current()";
```

Comment:
Again is this a `cppcheck` finding as the PR suggests, or something you felt was worth including on-top of the tooling suggestions? I would prefer to have a clean PR that's only applying the tooling bits, so that we can have a discussion and choice about including the tooling in the CI later.

Reply:
this is a fix for the `shadowed variable` warning which means there is another variable/function with the same name in the outer scope.